src/utils/colors.js

```diff
@@ -156,6 +156,13 @@ export function mapCategoryColorNameToStyles(colorName: string): ColorStyles {
         selectedTextColor: '#fff',
         gravity: 7,
       };
+    case 'magenta':
+      return {
+        selectedFillStyle: MAGENTA_60,
+        unselectedFillStyle: MAGENTA_60 + '60',
+        selectedTextColor: '#fff',
+        gravity: 7,
```

Comment:
Should we keep 7 for the gravity here? I think we can also put it to the further back, but don't have a strong opinion on it.

Reply:
So, I thought about this, and realized I have no idea :D I believe some of the existing gravities are quite bad but didn't want to change them all. That's where my thinking about moving away the gravity value into the category information in the profile metadata came from too... it seems a bad design decision that the gravity is tied to colors rather than categories... So I decided to... leave it.
I'm happy to put a bigger value if you think it's better!

Reply:
I used 8 and increased all the ones below.

Reply:
Works for me!